css/game.css

```diff
@@ -22,11 +31,49 @@ img.missile {
     margin: 0 auto;
 }
 
-
-img.city {
+.city {
     width: 183px;
+    height: 600px;
     display: block;
+    position: absolute;
+    top: 0;
+    left: 14%;
```

Comment:
this is a code smell. let the citys centre themselves using margin then make the missiles centre using left and transform